assets/js/components/notifications/notification.js

```diff
@@ -146,6 +146,8 @@ class Notification extends Component {
 			module,
 			moduleName,
 			pageIndex,
+			pageSpeedLink,
+			pageSpeedLabel,
```

Comment:
These props should be less specifically named since the component is generic, for example `anchorLink`, `anchorLinkLabel`.

Reply:
Added as requested here and in `dashboard-setup-alerts.js`